src/gui/macutilities.mm

```diff
@@ -100,4 +107,14 @@ void openFiles(const QSet<QString> &pathsList)
             [[NSWorkspace sharedWorkspace] activateFileViewerSelectingURLs:pathURLs];
         }
     }
+
+    QString badgeLabelText()
```

Comment:
Those 2 functions are obsoleted in Qt docs: https://doc.qt.io/qt-5/qtmac-obsolete.html#setBadgeLabelText
If you copied the implementations from Qt then the obsoleted marking should carry along too.

Reply:
>Those 2 functions are obsoleted in Qt docs: https://doc.qt.io/qt-5/qtmac-obsolete.html#setBadgeLabelText

Those 2 functions are **obsoleted in Qt** (apparently, just because they're going to delete them later). This does not mean at all that we should mark our similar functions as "obsolete". This marking generally makes little sense for internal application functions (as opposed to public/exported library functions).
>If you copied the implementations from Qt

They are not formally copied from there. I only took Qt's advice on how to replace them.
Even if I copied them, their contents are too trivial to classify this copying as borrowing, rather than, say, just being lazy to write it by hand.

Reply:
>Those 2 functions are obsoleted in Qt (apparently, just because they're going to delete them later). 

The real question is why they are not maintaining it anymore? is it obsolete at the system level or something else?

Reply:
>is it obsolete at the system level or something else?

I didn't find any mention of it.
>why they are not maintaining it anymore? 

Apparently, they are too trivial for developers to puzzle over where to put them after the abolition of Qt Mac Extras module. Qt just recommends using NSApplication properties directly instead of this wrappers from Qt Mac Extras (even in Qt5).